ReactiveCocoaTests/Swift/SignalLifetimeSpec.swift

```diff
@@ -26,37 +26,41 @@ class SignalLifetimeSpec: QuickSpec {
 				expect(signal).to(beNil())
 			}
 
-			it("should deallocate even if it has an observer") {
+			it("should deallocate if it does not have any observers") {
```

Comment:
I think we should write another version of this that makes the `signal` be initially retained, and verify that it is not deallocated, but then it is once the last strong reference is removed.
It should be equivalent to this one, but since the timing is slightly different, it's worth having.